documentation/src/main/asciidoc/topics/config_examples/server_network_matchhost.xml

```diff
@@ -1,5 +1,6 @@
+<!-- Selects an IP address that is assigned to a matching host name. -->
 <interfaces>
   <interface name="public">
-    <match-host value="my_host_name"/>
+    <match-host value="my_host_*"/>
```

Comment:
`_*` mathes zero or more `_`. I guess you want `my_host\..*`

Reply:
reverted. thanks for catching it.